test/UnitTests/CheckoutControllerTest.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+using Xunit;
+using MusicStore.Controllers;
+using MusicStore.Models;
+
+namespace UnitTests
+{
+    public class CheckoutControllerTest
+    {
+        IServiceProvider _serviceProvider;
+
+        public CheckoutControllerTest()
+        {
+            var services = new ServiceCollection();
+
+            services.AddEntityFramework()
+                      .AddInMemoryStore()
+                      .AddDbContext<MusicStoreContext>();
+
+            _serviceProvider = services.BuildServiceProvider();
```

Comment:
That's my understanding after discussing with @lukewaters. The model is defined in the MusicStoreContext class. I haven't thought about its lifetime though. 
@lukewaters, could you add a little bit explains here??